tests/fixtures/noqa/noqa.py

```diff
@@ -747,3 +747,17 @@ def my_function():
 
 def pos_only_problem(first_argpm=0, second_argpm=1, /):  # noqa: WPS475
     my_print(first_argpm, second_argpm)
+
+
+TypeVarDefault = TypeVar("T", default=int)
```

Comment:
You would need to create a new file called `noqa312.py` and ignore it for all versions prior to 3.12